Telepathy/Client.cs

```diff
@@ -22,7 +22,7 @@ public bool Connected
             }
         }
 
-        public bool NoDelay = true;
+        public bool NoDelay = false;
```

Comment:
why false by default?

Reply:
it will be true by default,   WIP at the moment,  don't merge.